### DISCUSSION TITLE
Taxjar Customer Exemption

### DISCUSSION THREAD

**Discussion by jaymar-webriq** [swellstores/community#381] [Help] [answered]
Hello guyz, just need some help or someone integrated a taxjar exemption using specific customer?
We have create a custom field for enabling customer that is exempted to a tax in checkout, the problem is how enable this in taxjar.
We already tried to add a customer info like swell customer id and encode it to taxjar customer exemption, but nothings happen.

Thank you

**Comment by sags95** [comment]
@jaymar-webriq Unfortunately this is a bit outside of what is supported by default with the Taxjar integration. We don't send customer details beyond the shipping and billing addresses during the tax calculation request, but let me check with the team to see if there are alternative solutions.

**Reply by jaymar-webriq** [reply, marked as answer]
hello @sags95,

swell support sends me a workaround solution for this situation, they suggest to add this cart field `taxes_fixed:true` to skip tax calculation, this thing works for us, thanks for the reply
